### ISSUE TITLE
Word boundary errors possible

### ISSUE THREAD

**Issue by panathea** [JoshuaGrams/steno-jig#24]
While doing the single-syllable exercise, I've run into word boundary errors, such as:

- `Kass set` → `cassette`
- `pup pet` → `puppet`

It would be nice if the lesson was aware of these entries and never suggested "pup pet" or "Kass set". Maybe more generally, all random lessons could be checked for word boundary errors.

I'd be willing to work through a solution and make a PR if you are open to it.

**Comment by JoshuaGrams** [comment]
Yeah, and car rot. I have a list of them somewhere. I wasn't sure how to handle that: would we need the whole Plover dictionary to find them? I was reluctant to slow the loading more by adding another even bigger data file, but I suppose most people use it enough that it's just in cache all the time.  And if Twitter can ask you to load 9MB of crap to read a 140-character message, then I suppose it's OK to ask you to load three or four to do something useful.

But yeah, go for it.

**Comment by panathea** [comment]
I was thinking that maybe we brute force and look up all the combinations
in main.json and build a small word boundaries dictionary in order to not
hold the whole dictionary.

**Comment by JoshuaGrams** [comment]
Ah, that could work.
